js/dcf-tabs.js

```diff
@@ -0,0 +1,111 @@
+class DCFTabs {
+  constructor(tabGroups) {
+    this.tabGroups = tabGroups;
+  }
+  // Tab switching function
+  switchTab = (oldTab, newTab, tabs, panels) => {
```

Comment:
Getting lint error : `6:13  error  Parsing error: Unexpected token =`
Update to `switchTab(oldTab, newTab, tabs, panels) {` to fix and match format of initialize